app/definitions/IMessage.ts

```diff
@@ -1,11 +1,25 @@
 import Model from '@nozbe/watermelondb/Model';
 import { MarkdownAST } from '@rocket.chat/message-parser';
 
+import {
+	MESSAGE_TYPE_LOAD_MORE,
+	MESSAGE_TYPE_LOAD_NEXT_CHUNK,
+	MESSAGE_TYPE_LOAD_PREVIOUS_CHUNK
+} from '../constants/messageTypeLoad';
 import { IAttachment } from './IAttachment';
 import { IReaction } from './IReaction';
 import { IUrl } from './IUrl';
 
-export type MessageType = 'jitsi_call_started' | 'discussion-created' | 'e2e' | 'load_more' | 'rm' | 'uj';
+export type MessageType =
+	| 'jitsi_call_started'
+	| 'discussion-created'
+	| 'e2e'
+	| 'load_more'
+	| 'rm'
+	| 'uj'
+	| typeof MESSAGE_TYPE_LOAD_MORE
+	| typeof MESSAGE_TYPE_LOAD_NEXT_CHUNK
+	| typeof MESSAGE_TYPE_LOAD_PREVIOUS_CHUNK;
```

Comment:
The last thing I would "like" would be to remove this type here from the const and leave it as an interface or enum.
If you want to call Diego, but I think it's better, because little by little the dependence on fixed constants is removed.

Reply:
Nice, good point! I will refactor this